Aptoide/aptoide-client-v8
# Feature/an 1698 timeline ads

What does this PR do?

   This pull-request adds ads in the Timeline using twitter MoPub integration.


Where should the reviewer start?

How should this be manually tested?

  Open Aptoide v8 > scroll timeline > check ads

What are the relevant tickets?

Tickets related to this pull-request: [AN-1698](https://aptoide.atlassian.net/browse/AN-1698)

Screenshots (if appropriate):
N/A

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
